v2/concordium/types.proto

```diff
@@ -1945,6 +1945,15 @@ message ConsensusInfo {
   BlockHash current_era_genesis_block = 29;
   // Time when the current era started.
   Timestamp current_era_genesis_time = 30;
+  // The current duration to wait before a round times out. Present from protocol version 6.
+  optional Duration current_timeout_duration = 31;
+  // The current round. Present from protocol version 6.
+  optional uint64 current_round = 32;
```

Comment:
It would be more consistent to define a new message `Round`.

This is what we do for other "newtypes", such as Slot, Epoch.